controllers/commentsController.js

```diff
@@ -15,4 +21,27 @@ const getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
-module.exports = getCommentsByArticleId;
+
+const postCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  const promises = [
+    selectArticleById(article_id),
```

Comment:
Whilst this is great, this should be refactored at a later date as you are making 3 requests to the database when we don't need to. The author column and article_id column are foreign keys and if they don't correspond with a valid value, PSQL will throw an error.